TAXES_IN_NORWAY.md

```diff
@@ -0,0 +1,102 @@
+# Taxes
+
+So it's awesome to make money doing stuff you love, however some people are
+not so lucky. That is basically why we pay so much tax in Norway,
+to help others that are not as fortunate as your self.
+
+However we also get allot back from those taxes as well. For example
+free health care, free education, financial security, etc. and thus
+a more stable society to live in. Since, you know, you won't loose your
+house and live on the street by the first sign of trouble. Yay!
+
+Happy programmer:
+
+![Happy programmer](http://www.rdegges.com/images/2012/programming.gif)
+
+## Different types of taxes
+
+### Regular tax
+
+Varies between 11% and 47% based on you level of income. there are also
+some deductions from this percentage based on your mortgage, number of
+kids, type of work, etc.
+
+[See a table of tax percentages based on yearly income here](http://www.smartepenger.no/skatt/103-skatt/653-skatteprosenter-pa-lonnsinntekt).
+Sadly this is only available in Norwegian, but google
+translate might help you understand it. The first table is without
+mortgage loan and the second is with an estimated mortgage of 2 times
+your income (a modest loan if you live in Oslo).
+
+Regular taxes are held back and paid in by your employer and is part of
+an automatic process strictly regulated by Norwegian accounting principles.
+
+### Payroll tax
+
+**Applies to**: The Employer
+
+**Percentage**: Between 14.1% and 0%. 14.1% for Hyper since our
+employees are based in Oslo.
+
+Why should employers have to pay the government to hire you? Basically
+this works like an insurance for you as an employee. If you are fired
+and cannot find another job in Norway you will still receive a salary for
+up to 2 years (minumum of 1 year). The salary is based on your income
+the last 3 years and is less a bit less than what you usually would
+earn working.
+
+There are more reasons for employers paying Payroll tax but i will not
+go into them here.
+
+### Vacation money (Feriepenger)
+
+**Applies to**: Full time employees
+
+**Percentage**: minimum 10.2 % or more. or 12.5% for employees
+of age 60 or above.
+
+#### The good part
+
+So first of all, this is not a tax. It is accounting law. Vacation money
+is a sum of money saved on your behalf by your employer to ensure that you
+have a salary that equals your average monthly income during vacation.
+
+This is good because we have so many vacation days in Norway (yay!).
+A minimum of 4 week (counted as 4 * 5 workdays, so 20 work days in total).
+**At Hyper we are even luckier and have 5 vacation weeks.**
+
+So basically if you earn 110.000 Kr. in a year. Instead of receiving
```

Comment:
hmmm actually i used 110.000 here to simplify the example since we get paid approximately 11 months of the year (hence 10.000 kr each month). However this example is incorrect as you probably noticed. 

Its supposed to be like so: Out of 120.000 kr. we get 107.760 kr. paid out directly thought the hours we work. The rest is saved for us by the employer and given to us during the time we are on vacation. This is usually set to a fixed month, like every June.